cvm/quantization/from_tensorflow.py

```diff
@@ -592,10 +633,15 @@ def convert_tfnode(tfnode, graph, params, infer_shapes, logger=logging):
             elif k not in ('dtype', '_output_shapes', '_class'):
                 raise NotImplementedError \
                     ("Other attributes for a Const(param) Node {} ? .".format(k))
-    elif op_name in ['Placeholder', 'PlaceholderWithDefault']:
+    elif op_name in ['Placeholder', 'PlaceholderWithDefault', 'FIFOQueueV2']:
         input_shape = \
             tensor_util.tensor_shape_proto_to_list(attr['shape'].shape)
-        dtype = tensor_util.tensor_type_to_numpy(attr['dtype'].type)
+        if not input_shape:
+            input_shape = default_input_shape
```

Comment:
Cannot the input_shape be extracted from the attr['shape'], or the shape is not indicated in the TensorFlow model?

I suggest that the input shape should be passed through the parameters list with user-settings, instead of setting default shape.

Reply:
already fixed